src/shared.rs

```diff
@@ -1,17 +1,26 @@
-use std::collections::BTreeMap;
-
-use flux::semantic::types::{Function, MonoType};
 use lspower::lsp;
 
-use crate::shared::all_combos;
+pub fn get_package_name(name: &str) -> Option<String> {
+    let items = name.split('/');
+    items.last().map(|n| n.to_string())
```

Comment:
Looks like we used `String::from` in `get_argument_names` below would we want to use that here too to be consistent?

Reply:
Yeah, I'm not sure. There are probably eleven different ways to convert to a string, and I don't think I want to over-prescribe here, for the reason we talked about in DM. My personal preference is using `.into()`, but I think it might be a bit bikeshed to specify this.

Reply:
Oh! This patch looked weird to me, because I didn't _think_ I created any new code in this patch. Git is confused at the merging of some submodules. This function specifically is just a move from another file. In that case, I'd really prefer not to touch it in this patch, because I'd rather separate the move from the change. Otherwise, the review gets difficult.